packages/botonic-cli/src/commands/train.ts

```diff
@@ -1,44 +1,103 @@
 import colors from 'colors'
-import * as path from 'path'
+import { existsSync } from 'fs'
+import { join } from 'path'
 
 import { spawnNpmScript, track } from '../utils'
 
+enum TASKS {
+  NER = 'ner',
+  TEXT_CLASSIFICATION = 'text-classification',
+}
+
 export default class Run extends Command {
-  static description = 'Train your bot with NLU'
+  static description = 'Train your bot with NLP'
 
   static examples = [
-    `$ botonic train
-    TRAINING MODEL FOR {LANGUAGE}...
+    `$ botonic train [--task=<${Object.values(TASKS).join('|')}>]
+    TRAINING MODEL...
     `,
   ]
 
   static flags = {
-    lang: flags.string(),
+    task: flags.string(),
   }
 
   static args = []
 
   async run(): Promise<void> {
-    const botonicNLUPath: string = path.join(
-      process.cwd(),
-      'node_modules',
-      '@botonic',
-      'nlu'
-    )
-    try {
-      await import(botonicNLUPath)
-    } catch (e) {
-      console.log(e)
-      console.log(
-        colors.red(
-          `You don't have @botonic/plugin-nlu installed.\nPlease, install it with the following command:`
-        )
+    const { flags } = this.parse(Run)
+    const task = flags.task
+    if (task) {
+      this.runSpecificTask(task)
+    } else {
+      this.runAllTasks()
+    }
+  }
+
+  private runAllTasks(): void {
+    this.runNerTask()
+    this.runTextClassificationTask()
+  }
+
+  private runSpecificTask(task: string): void {
+    switch (task) {
+      case TASKS.NER:
+        this.runNerTask()
+        break
+      case TASKS.TEXT_CLASSIFICATION:
+        this.runTextClassificationTask()
+        break
+      default:
+        this.logInvalidTask(task)
+        break
+    }
+  }
+
+  private runNerTask(): void {
+    const packageName = `plugin-${TASKS.NER}`
+    if (this.isPackageInstalled(packageName)) {
+      track(`Trained with Botonic train:${TASKS.NER}`)
+      spawnNpmScript(`train:${TASKS.NER}`, () => `Finished training `)
+    } else {
+      this.logPackageNotInstalled(packageName)
+    }
+  }
+
+  private runTextClassificationTask(): void {
+    const packageName = `plugin-${TASKS.TEXT_CLASSIFICATION}`
+    if (this.isPackageInstalled(packageName)) {
+      track(`Trained with Botonic train:${TASKS.TEXT_CLASSIFICATION}`)
+      spawnNpmScript(
+        `train:${TASKS.TEXT_CLASSIFICATION}`,
+        () => `Finished training `
       )
-      console.log(`$ npm install @botonic/plugin-nlu`)
-      return
+    } else {
+      this.logPackageNotInstalled(packageName)
     }
-    track('Trained with Botonic train')
-    spawnNpmScript('train', () => 'Finished training')
+  }
+
+  private logInvalidTask(task: string): void {
+    console.log(
+      colors.red(
+        `Unsupported task '${task}'. Available tasks: '${Object.values(
+          TASKS
+        ).join("', '")}'.`
+      )
+    )
+  }
+
+  private isPackageInstalled(packageName: string): boolean {
+    const path = join(process.cwd(), 'node_modules', '@botonic', packageName)
```

Comment:
how do you know that cwd will be the top folder here?

Reply:
You're right, the code assumes that the user runs the `botonic train` command from the bot root folder. 
I would need to look at the node_modules of the bot, but there is an easy way of doing this?

Reply:
If you rebase to master, you'll see that now in https://github.com/hubtype/botonic/blob/master/packages/botonic-cli/src/constants.ts#L4 we have constant BOTONIC_PROJECT_PATH
please do it after all these PRs are merged